extensions/objects/templates/int-editor.component.ts

```diff
@@ -23,17 +23,24 @@
 import { Component, EventEmitter, Input, OnInit } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 
+interface IntValidatorSchema {
+  type: string,
+  validators: { id: string }[]
```

Comment:
Should you use `Record` here?

Reply:
We would still have to declare an interface if we use `Record<,>` here. We can do `validators: Record<string, string>[]` but not `validators: Record<id, string>[]`. 
As can be seen in [record type](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype)